_people/hycinth-umaran.markdown

```diff
@@ -5,8 +5,10 @@ Photo: "/uploads/mumaran-23a31f.jpg"
 Member Type:
   Is Staff: true
 Job Title: Scrum Master and Agile Coach
+Country: Philippines
+Team: Tech Team
```

Comment:
is the Tech Team supposed to be its own category on the staff page?

Reply:
@dakotabenjamin my idea is to open the possibility to subdivide the Global staff by teams later, if desired, or show that information on the user page.

Reply:
I thought that might be the case. I like the idea.